usr/lib/systemd/system-preset/50-playtron.preset

```diff
@@ -1,3 +1,4 @@
 enable lightdm.service
 enable create-swap.service
 enable resize-root-file-system.service
+disable firewalld.service
```

Comment:
Disable the `firewalld` service here instead of in the kickstart file so that factory reset doesn't re-enable it.